search/includes/classes/class-queue.php

```diff
@@ -612,11 +635,10 @@ public function checkout_jobs( $count = 250 ) {
 		$table_name = $this->schema->get_table_name();
 
 		// TODO transaction
-		// TODO only find objects that aren't already running
 
 		$jobs = $wpdb->get_results(
 			$wpdb->prepare(
-				"SELECT * FROM {$table_name} WHERE ( `start_time` <= NOW() OR `start_time` IS NULL ) AND `status` = 'queued' LIMIT %d", // Cannot prepare table name. @codingStandardsIgnoreLine
+				"SELECT * FROM {$table_name} WHERE ( `start_time` <= NOW() OR `start_time` IS NULL ) AND `status` = 'queued' ORDER BY `job_id` LIMIT %d", // Cannot prepare table name. @codingStandardsIgnoreLine
```

Comment:
Mentioned above, it does seem the `prepare` is used and is the safer approach.

Reply:
Same as above, we still inline the table name, which can't be prepared, yet the coding standard complains if you have any inline variable in the query string.